src/main/java/org/apache/commons/lang3/StringUtils.java

```diff
@@ -6654,10 +6656,8 @@ private static String replaceEach(
 
         // make sure lengths are ok, these need to be equal
         if (searchLength != replacementLength) {
-            throw new IllegalArgumentException("Search and Replace array lengths don't match: "
-                + searchLength
-                + " vs "
-                + replacementLength);
+            throw IllegalArgumentExceptions.format("Search and Replace array lengths don't match: %s vs %s",
+                    searchLength, replacementLength);
```

Comment:
%s => %d ?

Reply:
I will use `"Search and Replace array lengths don't match: %,d vs %,d"`